src/main/java/core/basesyntax/strategy/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (transaction.getQuantity() < 0) {
+            throw new RuntimeException("Balance must be positive! " + transaction.getQuantity());
+        } else {
+            FruitStorage.getStorage().put(transaction.getFruit(), transaction.getQuantity());
+        }
```

Comment:
It'll be better if you create a local variable here for transaction.getQuantity, so all these method calls won't affect on memory

Reply:
we need to separate imports and class with empty line(mvn clean package requests)'CLASS_DEF' should be separated from import.